rdflib/namespace.py

```diff
@@ -199,6 +199,7 @@ class ClosedNamespace(Namespace):
 
     def __new__(cls, uri, terms):
         rt = super().__new__(cls, uri)
+        rt.__terms = terms
```

Comment:
Is this used?
```suggestion
```

Reply:
Apologies, I was thinking of fixing __repr__ to show terms, but then decided against that, removed it.

Reply:
Interesting. The `terms` would be needed to reconstruct the `ClosedNamespace` object, but the full list might indeed be unruly for printing to stdout. :thinking:

Reply:
I think it is best to not try and fix all problems there in one PR, better to get it in to the release and make an issue for the other issues. This is not a problem introduced in your PR.